.github/workflows/tpp-benchmark.yml

```diff
@@ -25,12 +28,16 @@ env:
 
 jobs:
   Check_LLVM:
+    if: ${{ github.event_name }} == "push" || \
+        (${{ github.event_name }} == "pull_request" && ${{ github.event.label.name == 'benchmark' }})
     uses: ./.github/workflows/tpp-llvm.yml
     secrets: inherit
 
   TPP-MLIR-SPR-BASE:
     runs-on: pcl-tiergarten
-    if: ${{ github.event_name }} == "push" || ${{ github.event_name }} == "pull_request" || ${{ inputs.RUN_SPR_BENCH }} == 1
+    if: ${{ github.event_name }} == "push" || \
+        ${{ inputs.RUN_SPR_BENCH }} == 1 || \
```

Comment:
With the current conditional, does `${{ inputs.RUN_SPR_BENCH }} == 1` even control anything?

Reply:
Yup. This works for the `workflow_dispatch`.